GameEngine/EngineInternal.cs

```diff
@@ -34,8 +34,12 @@ public void StartEngine()
         {
             // We can't get the real PlayerCharacter because the game isn't loaded yet.
             // But we can fake it out for this call
-            var playerCharacter = new PlayerCharacter("loader", 1);
-            InternalCommandHelper.TryRunInternalCommand("load", new List<string>(), this, playerCharacter);
+            var playerCharacters = new List<Character>()
```

Comment:
Can probably just change this to be a "loaderCharacter" vs new var playerCharacters below